src/ci/docker/host-x86_64/x86_64-gnu-tools/browser-ui-test.version

```diff
@@ -1 +1 @@
-0.8.4
\ No newline at end of file
+0.8.5
```

Comment:
Is there some feature added in a new version of browser-ui-test that this PR needs?

Reply:
No, it's just for personal convenience. It warns when the versions mismatch. I can remove this commit though if you prefer.

Reply:
No, it's fine. It looks like the only change is [this noSandbox change](https://github.com/GuillaumeGomez/browser-UI-test/pull/297).

Reply:
It was bugfix I needed because I'm working on docs.rs GUI tests (I need to resume my work there).